docs/UserGuide.md

```diff
@@ -211,44 +244,58 @@ In order of priority, the planner will prioritise `Events`, then `Deadlines`, th
 Examples:
 - `plan 5` will plan your month according to an ideal effort level of 5 per day.
 
-### Clearing all entries : `clear`
+### 2.11 Clearing all entries : `clear`
 
 Clears all entries from the address book.
 
 Format: `clear`
 
-### Exiting the program : `exit`
+### 2.12 Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
-### Saving the data
+## 3. Storage 
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+### 3.1 Saving the data
 
-### Editing the data file
+TaskBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+### 3.2 Editing the data file
+
+TaskBook data are saved as a JSON file `[JAR file location]/data/taskbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, taskBook will discard all data and start with an empty data file at the next run.
 </div>
 
-### Archiving data files `[coming in v2.0]`
+### 3.3 Archiving data files `[coming in v2.0]`
 
 _Details coming soon ..._
 
 --------------------------------------------------------------------------------------------------------------------
 
-## FAQ
+## 4. FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
+### Q1: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
 
+### Q2: What is a valid Date?<br>
+**A**: A valid Date must be in the format of `YYYY-MM-DD HHMM` such as `2023-07-13 1800` (13 July 2023, 6PM). A valid Date must have a valid time. A minimum of 4 characters must be supplied and only a maximum of 4 character will be parsed
+  e.g. `2023-07-13 180` is invalid and `2023-07-13 18000000000` is understood as `2023-07-13 1800`
```

Comment:
Would it be better to clarify that we accept 2 types of date formats? The distinction between Date and LocalDate may not be clear to users!